### PR TITLE
Markdown chores: NOTE rendering, trailing spaces and multiple empty lines fix

### PR DESCRIPTION
**Pull request by Andrei-Aksionov**
Hi there 👋 

I've noticed that in some markdown files [!NOTE] label is not rendered correctly because of the appended characters.
Also, I standardized all "note" labels to be only [!NOTE] and not [!NOTE]\\.
Fixed trailing white spaces and removed unnecessary empty lines except those that Sebastian added lately: don't understand why he likes so much that many empty spaces but feel like if I delete them - something is gonna happen to me 😆 🙀.

---- 

@rasbt if you're reading this, in VSCode: 	
```text
"files.trimTrailingWhitespace": true,
```
😉😉

### CODE REVIEW

File: tutorials/neurips_challenge_quickstart.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ These don't include models that have been finetuned or otherwise aligned, as per
 | Models in Lit-GPT         | Reference                                                    |
 | ------------------------- | ------------------------------------------------------------ |
 | Meta AI Llama 2 Base      | [Touvron et al. 2023](https://arxiv.org/abs/2307.09288)      |
-| TII UAE Falcon Base       | [TII 2023](https://falconllm.tii.ae/)     
+| TII UAE Falcon Base       | [TII 2023](https://falconllm.tii.ae/)
```
Comment (carmocca) [inline tutorials/neurips_challenge_quickstart.md:37]: This table was missing the rightmost "|"?

GitHub seems to render it correctly anyways

**Review by carmocca — APPROVED** [review]
Thanks!

File: tutorials/neurips_challenge_quickstart.md (hunk 1 above)
Comment (Andrei-Aksionov) [reply tutorials/neurips_challenge_quickstart.md]: Eagle eye 😄.
Yes, indeed, it's missing.
GitHub does a lot of things behind the curtains to help users, and it's great, but I don't like to rely on it too much.
Fixed.

**Comment by rasbt** [comment]
Thanks @Andrei-Aksionov 

>  don't understand why he likes so much that many empty spaces

Haha, I often use a different markdown editor (not VS Code) for writing, and I think that's when this happens.

**Pull request merged**